### PR TITLE
Remove merchant preference language

### PR DESCRIPTION
**Pull request by ianbjacobs**
Redoing previous PR that I accidentally killed:
 https://github.com/w3c/browser-payment-api/pull/506#issue-220225305

This version takes into account the previous editorial suggestions (and tidy).

(Apologies for not doing this on a separate branch. In my rush to recover the
lost commit history I neglected to create a separate branch.)

This PR does not address feedback from @mattsaxon and @mountainhippo;
I am merely recreating what I deleted in error.

Ian

***
[Preview](https://s3.amazonaws.com/pr-preview/ianbjacobs/browser-payment-api/gh-pages.html) | [Diff](https://s3.amazonaws.com/pr-preview/w3c/browser-payment-api/df83baf...ianbjacobs:de74e6a.html)

### CODE REVIEW

**Review by marcoscaceres — APPROVED** [review]
Carrying over previous approval.

**Comment by marcoscaceres** [comment]
@ianbjacobs, can you please rebase?

**Comment by zkoch** [comment]
This reflects a WG decision, so we are going to merge once conflict is resolved.

**Comment by ianbjacobs** [comment]
Well, I seem to have failed at the rebase. Let me try again.

**Comment by ianbjacobs** [comment]
Ok, let me know if now ok.
Ian

**Review by marcoscaceres — APPROVED** [review]
Lgtm

**Comment by mattsaxon** [comment]
Copy and paste of my previous comments....

With the revised spec text;
150 + Enable user agents to streamline the users's payment experience
151 + by taking into account user preferences, merchant information,
152 + security considerations, and other factors.

Whilst it is clear to me how the other factor are exposed to the user-agent, there seems to me to be no way of the merchant expressing their preference.

I propose we make explicit that the ordering of the PMIs is that expression, but that we make no statement about the behaviour of the user-agent with regard to this ordering.
